rofeeah_ayeni/alpha_challenges/string_manipulation.md

```diff
@@ -19,13 +19,14 @@ iverson_rant = "If I can't practice, I can't practice, man. If I'm hurt, I'm hur
 Programmatically manipulate `iverson_rant` to reflect the output above.
 
 ```ruby
-# write your code here
+puts iverson_rant.split.map(&:chr).join
 ```
 
 2. Programmatically find out exactly how many times Iverson used the word 'practice' in his rant.
 
 ```ruby
-# write your code here
+puts iverson_rant.split("practice").count
+#not sure about this 
```

Comment:
There's an easy way to check! When I run your code, I get a count of 11. But if I manually count, I get 10. Your code is close, but it isn't _quite_ doing what you want it to do.